### PR TITLE
add all the style

### PR DESCRIPTION
**Pull request by Alzubair98**
i have made the styles before making the API 
i think its okay 
all the style is done 
thanks

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -166,3 +168,43 @@ body {
...
+.icon-continer {
+  position: relative;
+  left: 60rem;
+  top: 10px;
+  color: #277bc0;
+}
+
+.icon {
+  font-size: 2.5rem;
+}
```
Comment (DeliceLydia) [inline src/App.css:171]: - [ ] Styling should be EXACTLY the same as the one provided on the design [Zeplin](https://app.zeplin.io/styleguides), which means colors, and layouts. you can refer to this:
blue

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Alzubair98, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! Highlights for your project:
- Awesome README
- Followed git workflow


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/App.css (hunk 1 above)
Comment (Alzubair98) [reply src/App.css:171]: hi okay i will change the buttons and the colors thanks

**Review by Davidosky007 — APPROVED** [review]
Hi @Alzubair98 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* Well written PR description 👍 
* Good looking readme file 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**